includes/Core/Authentication/Google_Proxy.php

```diff
@@ -74,6 +74,55 @@ public function url( $path = '' ) {
 		return $url;
 	}
 
+	/**
+	 * Gets site fields.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	public function get_site_fields() {
+		return array(
+			'name'       => wp_specialchars_decode( get_bloginfo( 'name' ) ),
+			'url'        => home_url(),
+			'admin_root' => admin_url(),
+			'action_url' => admin_url( 'index.php' ),
+			'return_url' => $this->context->admin_url( 'splash' ),
```

Comment:
This is actually a change in requirements, but can you call these parameters `action_uri` and `return_uri` instead? It's more consistent with our other similar `redirect_uri` parameter for OAuth.

`url` should remain as is, also as it's used in a different way.

Reply:
Done!